src/screens/benefit/Details.tsx

```
@@ -199,40 +209,88 @@ const BenefitsDetails: React.FC = () => {
 		);
 	};
 
-	const extractRequiredDocs = (resultItem) => {
+	const extractRequiredDocs = (resultItem: any): DocumentItem[] => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)